### PR TITLE
Simplify GetSourcePath, remove unused code

### PR DESCRIPTION
**Pull request by dimitry-**

### CODE REVIEW

File: OrbitCore/Path.cpp [hunk 1]
```diff
@@ -258,42 +258,16 @@ std::string Path::GetAppDataPath() {
...
-}
-
-std::string Path::GetExternalPath() {
-  if (is_packaged_)
-    return GetExecutablePath() + "external/";
-  else
-    return GetSourceRoot() + "external/";
+static std::string GetSourceRoot() {
+  std::string currentDir = Path::GetDirectory(__FILE__);
+  return Path::GetParentDirectory(currentDir);
```
Comment (dpallotti) [inline OrbitCore/Path.cpp:263]: This assumes that Path.cpp is only one level below the source root, right?

Comment (dpallotti) [reply OrbitCore/Path.cpp:263]: Wasn't the previous implementation trying to be clever, although still relying on the very uncertain assumption that "bootstrab-orbit" be in the source root?

Comment (dimitry-) [reply OrbitCore/Path.cpp:263]: yes - added comment about it.

File: OrbitCore/CMakeLists.txt [hunk 2]
```diff
@@ -213,6 +213,7 @@ target_sources(OrbitCoreTests PRIVATE
     RingBufferTest.cpp
     StringManagerTest.cpp
     LinuxTracingSessionTests.cpp
+    PathTest.cpp
```
Comment (dpallotti) [inline OrbitCore/CMakeLists.txt:216]: I think you're missing a file in the commit.

Comment (dimitry-) [reply OrbitCore/CMakeLists.txt:216]: right - fixed.

**Pull request merged**